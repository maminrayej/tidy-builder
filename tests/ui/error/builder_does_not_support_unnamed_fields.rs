#[derive(tidy_builder::Builder)]
struct MyStruct(usize, usize);

fn main() {}
