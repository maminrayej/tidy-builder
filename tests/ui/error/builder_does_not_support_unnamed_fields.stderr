error: Builder does not support unnamed fields
 --> tests/ui/error/builder_does_not_support_unnamed_fields.rs:2:16
  |
2 | struct MyStruct(usize, usize);
  |                ^^^^^^^^^^^^^^
